### ISSUE TITLE
pip-licenses scan NOTICE.* file as well

### ISSUE THREAD

**Issue by Hannah-Jiang** [raimon49/pip-licenses#66]
We are using pip-licenses to collect LICENSES of third party dependencies, which works well. Is it possible to add NOTICE files as well so both LICENSE and NOTICE are returned if available.

**Comment by raimon49** [comment]
@Hannah-Jiang Thanks for the feature request. I think this can of course be implemented additionally.

Maybe your request is about the behavior of the `--with-license-file` option. Please comment the CLI execution output you want. It helps me and other contributors implement the feature.

**Comment by Hannah-Jiang** [comment]
Thanks for replying. 
Yes, I am using `pip-licenses --with-license-file --format=json` and looking for a similar one for NOTICE, something like `pip-licenses --with-license-file --with-notice-file --format=json`. 

Output I am looking for is:

```
{
    "License": "MIT License",
    "LicenseFile": "xxx",
    "LicenseText": "xxx",
    "NoticeFile": "xxx"
    "NoticeText": "xxx"
    "Name": "soupsieve",
    "Version": "2.0"
}
```

Do we have a rough timeline for this issue?

Thanks,
Hannah

**Comment by raimon49** [comment]
Thanks for the detailed specifications.
Scanning and printing NOTICE files will be included in the next release as pip-licenses 2.2.0. We aim to release it in April 2020.

**Comment by raimon49** [comment]
@Hannah-Jiang Hi, I just shipped the new version 2.2.0.

You can use the `--with-notice-file` option.

**Comment by Hannah-Jiang** [comment]
Thank you very much. I will try it out soon!
